apis/metal3.io/v1alpha1/baremetalhost_webhook.go

```diff
@@ -30,14 +31,14 @@ func (r *BareMetalHost) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-//+kubebuilder:webhook:verbs=create;update,path=/validate-metal3-io-v1alpha1-baremetalhost,mutating=false,failurePolicy=fail,groups=metal3.io,resources=baremetalhosts,versions=v1alpha1,name=vbaremetalhost.kb.io
+//+kubebuilder:webhook:verbs=create;update,path=/validate-metal3-io-v1alpha1-baremetalhost,mutating=false,failurePolicy=fail,sideEffects=none,admissionReviewVersions=v1;v1beta1,groups=metal3.io,resources=baremetalhosts,versions=v1alpha1,name=vbaremetalhost.kb.io
```

Comment:
We don't have a v1 API.

Reply:
I think this is for `WebhookConfigurations`, currently we are using v1 and we support v1beta1 too with `admissionReviewVersions=v1;v1beta1`

Reply:
Oh, OK, it's versioning for the data that is posted to the webhook:
https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#request

Carry on then.